docs/_pages/installation.markdown

```diff
@@ -18,5 +18,5 @@ sidebar:
 Follow these instructions to [install and configure Esri software]({{ site.url }}{{ site.baseurl }}/install-tririga) to generate and serve indoor maps from CAD data.
 
 ### Install and configure TRIRIGA software
-
+=======
```

Comment:
The single line with a single hyphen is important for Jekyll to generate the HTML from a markdown file.  Put back the -.